helm_deploy/hmpps-incident-reporting-api/values.yaml

```diff
@@ -43,7 +43,8 @@ generic-service:
           return 401;
         }
   env:
-    JAVA_OPTS: "-Xmx4g"
+    JAVA_OPTS: "-Xmx4g -Xmx4g -XX:+HeapDumpOnOutOfMemoryError"
```

Comment:
is `-Xmx4g` there duplicated by mistake or by design?

Reply:
Mike, AFAICT you don't need to duplicate these values, the 2nd `-Xmx...` would override the 1st anyway, so you can just remove the first value in all the `JAVA_OPTS` I think.